src/test/java/com/google/api/codegen/gapic/testdata/csharp/csharp_library.baseline

```diff
@@ -1996,7 +2128,16 @@ namespace Google.Example.Library.V1.Samples
                 SeriesUuid = new SeriesUuid(),
             };
             PublishSeriesResponse response = libraryServiceClient.PublishSeries(request);
-            // FIXME: inspect the results
+            // testing writing bytes field.
+            using (Stream output = File.Create($"uuid_of_series_with_book_{response.BookNames[0]}.raw"))
+            {
+                response.SeriesUuid.SeriesBytes.writeTo(output);
+            }
+            // testing writing string field.
+            using (StreamWriter outputFile = new StreamWriter(Path.Combine(docPath, "WriteLines.txt")))
```

Comment:
Writing a string can be done:
```C#
using System.IO;
...
File.WriteAllText(Path.Combine(docPath, "WriteLines.txt"), response.SeriesUuid.SeriesString);
```
This will overwrite any existing file content.

Reply:
+1 `File.WriteAllText` ~ wonderfully clear and simple for code samples